driver/configure/Makefile.inc.in

```diff
@@ -1,7 +1,7 @@
 MODULE_MAKEFILE_DIR := $(shell dirname $(abspath $(lastword $(MAKEFILE_LIST))))
 
 # Run the module build.sh (wrapper for make) script with an empty environment, but PATH
-HAS_@CONFIGURE_MODULE@ := $(shell env -i PATH="$(PATH)" KERNELDIR="$(KERNELDIR)" sh $(MODULE_MAKEFILE_DIR)/build.sh ; echo $$?)
+HAS_@CONFIGURE_MODULE@ := $(shell env -i CC=$(CC) KBUILD_MODPOST_WARN=$(KBUILD_MODPOST_WARN) PATH="$(PATH)" KERNELDIR="$(KERNELDIR)" sh $(MODULE_MAKEFILE_DIR)/build.sh ; echo $$?)
```

Comment:
Pass also `CC`; it is set by driverkit build templates at `make driver` [invocation](https://github.com/falcosecurity/driverkit/blob/master/pkg/driverbuilder/builder/templates/vanilla.sh#L65) and must be correctly passed to configure modules makefiles.